plexapi/base.py

```diff
@@ -110,12 +198,12 @@ def _buildItemOrNone(self, elem, cls=None, initpath=None):
         except UnknownType:
             return None
 
-    def _buildDetailsKey(self, **kwargs):
+    def _buildDetailsKey(self, **kwargs: Any):
```

Comment:
```suggestion
    def _buildDetailsKey(self, **kwargs: Optional[Dict[str, Any]]):
```

Shouldn't kwargs be a dictionary?

I'm unsure the type of the value of the dict, so left that as `Any`

Reply:
is a shorthand, `**_` is automatically parsed as `Dict[str, Unknown]`. providing `Any` is for value as key and dict is understood